src/components/subscriptions/edit/EditSubAddons.js

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+
+import { mapSubAddonsPropsToValues } from "./formatters";
+import ids from "../ids";
+
+import { useTranslation } from "react-i18next";
```

Comment:
We should only import `useTranslation` from our `i18n` module, which in turn is importing it from `next-i18next`.

Importing `useTranslation` from `react-i18next` can actually cause conflicts with the translations setup by `next-i18next`. To help avoid this confusion, we can stick with importing from our own `i18n` namespace.
```suggestion
import { useTranslation } from "i18n";
```

Also, it looks like `useTranslation` is being imported from `react-i18next` in `src/components/subscriptions/edit/EditQuotas.js` and `src/components/subscriptions/edit/EditSubscription.js`, so those should be updated to import from `i18n` as well (sorry I didn't catch those sooner).

Reply:
Thank you, I will address this